protocols/rMATs.sh

```diff
@@ -0,0 +1,70 @@
+#MOLGENIS walltime=23:59:00 mem=4gb ppn=4
+
+#Parameter mapping
+#list sampleMergedBam
+#string sampleMergedBamExt
+#string tempDir
+#string tmpDataDir
+#string project
+#string externalSampleID
+#string intermediateDir
+#string strandedness
+#string sifDir
+#string rMATsOutputDir
+#string annotationGtf
+#string projectJobsDir
+#string project
+#string groupname
+#string tmpName
+#string logsDir
+
+
+makeTmpDir "${intermediateDir}"
+tmpSampleMergedDedupBam="${MC_tmpFile}"
+
+mkdir -p "${rMATsOutputDir}/${externalSampleID}/tmp"
+
+# create list of bam files from design
+
+if [ -f "${intermediateDir}/${externalSampleID}.B1.txt" ]
+then
+  rm "${intermediateDir}/${externalSampleID}.B1.txt"
```

Comment:
you can shortcut this by just doing:
rm -f "${intermediateDir}/${externalSampleID}.B1.txt", it will force rm if it exists